templates/outer-loop/Dockerfile

```diff
@@ -33,10 +33,14 @@ FROM {{.OL_UBI_IMAGE}}
 # 2a) Copy user defined shared resources 
 COPY --from=compile --chown=1001:0 /work/shared /opt/ol/wlp/usr/shared/
 
+RUN true
```

Comment:
This `RUN true` looks hackish... if we're essentially doing this just for the GH Actions env, could we put a placeholder comment here, and then find/replace before we run the docker build in the test env?   (Like we do  a sed substitution in the deploy yaml).